bindings/python/tests/test_parse.py

```diff
@@ -32,3 +32,75 @@ def get_parsed_info(tmp_path, file_contents):
     )
 
     return parsed_info
+
+
+def test_return_type_composition(tmp_path):
+    file_contents = ""
+    parsed_info = get_parsed_info(tmp_path=tmp_path, file_contents=file_contents)
+
+    assert type(parsed_info) is dict
+    assert type(parsed_info["members"]) is list
+    assert len(parsed_info["members"]) == 0
+
+
+def test_translation_unit(tmp_path):
+    file_contents = ""
+    parsed_info = get_parsed_info(tmp_path=tmp_path, file_contents=file_contents)
+
+    assert parsed_info["kind"] == "TRANSLATION_UNIT"
+    assert parsed_info["depth"] == 0
+    assert parsed_info["line"] == 0
+    assert parsed_info["column"] == 0
```

Comment:
This can be ignored as it doesn't really harm anyone, but iirc we don't really care about line and column.